scripts/setup/bootstrap.sh

```diff
@@ -167,8 +167,14 @@ EOF
 # bootstrap or run_in_build_env.sh can be executed in a build env
 _ORIGINAL_PW_ENVIRONMENT_ROOT="$PW_ENVIRONMENT_ROOT"
 
+# pigweed does not seem to handle pwd involving symlinks very well.
```

Comment:
What's the actual problem we're fixing here? If Pigweed doesn't tolerate having symlinks anywhere in PWD that sounds like a Pigweed bug.

Reply:
The problem is bootstrap failing.  Yes, it's basically a pigweed bug, but until they fix it this at least makes things work....